mailbox/cassandra/src/main/java/org/apache/james/mailbox/cassandra/CassandraMessageManager.java

```diff
@@ -44,9 +45,11 @@
 
     public CassandraMessageManager(MailboxSessionMapperFactory mapperFactory, MessageSearchIndex index, 
             MailboxEventDispatcher dispatcher, MailboxPathLocker locker, Mailbox mailbox, QuotaManager quotaManager, 
-            QuotaRootResolver quotaRootResolver, MessageParser messageParser, MessageId.Factory messageIdFactory) throws MailboxException {
+            QuotaRootResolver quotaRootResolver, MessageParser messageParser, MessageId.Factory messageIdFactory,
+            FetchBatchSizes fetchBatchSizes) throws MailboxException {
         super(mapperFactory, index, dispatcher, locker, mailbox, new UnionMailboxACLResolver(), 
                 new SimpleGroupMembershipResolver(), quotaManager, quotaRootResolver, messageParser, messageIdFactory);
+        setFetchBatchSizes(fetchBatchSizes);
```

Comment:
The attribute is from the extended abstract class and is private.
I don't want to put this attribute in the constructor as it will break other implementations.